build-conf/Cobol.properties

```diff
@@ -36,7 +36,7 @@ cobol_testcase_loadPDS=${hlq}.TEST.LOAD
 
 #
 # List the data sets that need to be created and their creation options
-cobol_srcDatasets=${cobol_srcPDS},${cobol_cpyPDS},${cobol_objPDS},${cobol_dbrmPDS}
+cobol_srcDatasets=${cobol_srcPDS},${cobol_cpyPDS},${cobol_objPDS},${cobol_dbrmPDS},${cobol_myfilePDS}
```

Comment:
Where cobol_myfilePDS is defined?  
I do not see it in build-conf/Cobol.properties. If not defined we may have an exception?

Reply:
Sorry, this was left over from a test.